_posts/2022-11-18-geoserver-2-22-0-released.md

```diff
@@ -0,0 +1,91 @@
+---
+author: Martha Nagginda
+date: 2022-11-18
+layout: post
+title: GeoServer 2.22.0 Release
+categories:
+- Announcements
+- Vulnerability
+tags:
+- Release
+release: release_222
+version: 2.22.0
+jira_version: 16867
+---
+
+GeoServer [2.22.0](/release/2.22.0/) release is now available with downloads ([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-bin.zip/download), [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-war.zip/download), [windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/GeoServer-2.22.0-winsetup.exe/download)), along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/geoserver-2.22.0-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.22.0/extensions/).
+
+This is a stable release of the GeoServer 2.22.x series, made in conjunction with GeoTools 28.0 
+and GeoWebCache 1.22.0.
+
+All major new features have been described in the [Release Candidate (RC) Blog post](https://geoserver.org/announcements/2022/10/19/geoserver-2-22-RC-released.html).
+
+Thanks to Martha Nagginda (GeoSolutions) and Andrea Aime (GeoSolutions) for making this release.
+
+### Improvements and Fixes
```

Comment:
Please use 2.22-RC release announcement as starting point (it includes visual reference of new features and functionality).

Reply:
@jodygarnett For the notes in the 2.22-RC, i linked it on line 21 mentioning a read of the major features, than copying them again. what do you think?

Reply:
I have added the notes from 2.22-RC as suggested.

Reply:
Perfect thank you - folks need the visual.